@xen-orchestra/web-core/lib/components/UiCard.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <div class="ui-card" :style="{ flexDirection: direction, padding: padding ? '2.4rem' : 0 }">
+    <slot />
+  </div>
+</template>
+
+<script lang="ts" setup>
+type Props = {
+  direction: 'column' | 'row'
+  padding: boolean
+}
+
+defineProps<Props>()
+</script>
+
+<style lang="postcss" scoped>
+.ui-card {
+  display: flex;
+  border-radius: 0.8rem;
+  box-shadow: var(--shadow-200);
```

Comment:
According to the design system, there is no shadow, but a border:
```suggestion
  border: 0.1rem solid var(--color-grey-500);
```